.github/workflows/rust.yml

```diff
@@ -101,3 +102,35 @@ jobs:
       - uses: Swatinem/rust-cache@v1
       - name: Doc
         run: RUSTDOCFLAGS="-Dwarnings" cargo doc --workspace --all-features
+
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    if: "startsWith(github.ref, 'refs/tags/')"
+    needs: [ check, test ]
+    environment: VALIDATOR_RELEASE
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: Fetch Substrait submodule tags
+        working-directory: substrait
+        run: |
+          git fetch --recurse-submodules=no origin +refs/tags/*:refs/tags/*
+          git describe --dirty --tags
+      - name: Local build to populate resource files
+        working-directory: rs
+        run: cargo build
```

Comment:
Oh, good catch, probably not. Forgot to copypaste the toolchain selection thing and didn't notice because, apparently, it worked without (I already tested all this logic in my fork).

Reply:
305a401